electron/electron.livepeer.js

```diff
@@ -18,9 +18,10 @@ const startLivepeer = (sender) => {
             global.sharedObj.livepeerProc = 'local';
             log.info('LivePeer is already running.');
         } else if (global.sharedObj.livepeerProc == null) {
-            const args = [];
+            const args = [
+                '-monitor', true,
+                '-monitorhost', 'http://viz.livepeer.org:8081/metrics'];
```

Comment:
yeah, because node expect `-monitor=true` not `-monitor true`, to make it work you can strip the true

```js
const args = [
    '-monitor',
    '-monitorhost', 'http://viz.livepeer.org:8081/metrics'];
```